gbpservice/nfp/modules/configurator.py

```diff
@@ -260,7 +260,9 @@ def get_notifications(self, context):
 
         """
 
-        notifications = self.sc.get_notification()
+        notifications = self.sc.get_stash_event()
+        if notifications:
+            LOG.info("GET NOTIFICATIONS API SENDING DATA: %r" % notifications)
```

Comment:
Fix this log statement and make it the way it is done in rest of the code. We can discuss.